features/tracker-allowlist.json

```
@@ -234,6 +234,17 @@
                     }
                 ]
             },
+            "analytics.analytics-egain.com": {
+                "rules": [
+                    {
+                        "rule": "https://analytics.analytics-egain.com/onetag/EG94020756",
```

```suggestion
                        "rule": "analytics.analytics-egain.com/onetag/",
```

We can leave the ID off in case that updates. Do you know if the `cloud-us` subdomain causes any issues?